src/widgets/helper/ChannelView.cpp

```diff
@@ -2079,6 +2079,9 @@ void ChannelView::handleLinkClick(QMouseEvent *event, const Link &link,
                 }
             }
         }
+        case Link::CopyToClipboard: {
+            crossPlatformCopy(link.value);
+        }
```

Comment:
You are missing a `break` statement for the new case. This means that the above case will fall through to this when clicking on a channel name in the mentions split, and that channel is no longer open in any tab.

(In that case, the clicked channel name will be copied to your clipboard.)

```suggestion
        }
        break;
        case Link::CopyToClipboard: {
            crossPlatformCopy(link.value);
        }
```

Reply:
Looks like I've forgotten about the `break` in that Pull Request. Will add :+1: